### PR TITLE
Added a Pull Request Template

### PR DESCRIPTION
**Pull request by KushalXCoder**
Based on the issue description and some online learning, I have added things that I thought would be required. But, still, review it and suggest any additions or deletions in it.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (Saumya40-codes) [inline .github/ISSUE_TEMPLATE/bug_report.md:1]: Please keep both bug_report and feature_request template as it is. No need to delete

**Comment by Saumya40-codes** [comment]
LGTM overall. Thanks

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (KushalXCoder) [reply .github/ISSUE_TEMPLATE/bug_report.md:1]: Added again.

**Comment by Saumya40-codes** [comment]
Please don't combine 2 issue fixes in one PR, might get hard to keep track going forward

**Comment by Saumya40-codes** [comment]
Just a suggestion
### This is irrelevant to this issue but might create problem with your pr going foward

You can go to your cloned/forked repo and click `Sync fork` option or `update branch` option  (make sure current branch is `main` there)

then 
```
 git checkout main
```

and 

```
git pull
```

Then you can create another branch to work on. This make sure you are working on latest codebase, might decrease the chance of merge conflict or any other deps issue

**Comment by KushalXCoder** [comment]
I didn't merge two issues, I just added back the bug_report and feature_request which I deleted earlier, and added them in the original folder in which they were, which I deleted at starting.

**Comment by Saumya40-codes** [comment]
> I didn't merge two issues, I just added back the bug_report and feature_request which I deleted earlier, and added them in the original folder in which they were, which I deleted at starting.

imo this is part of the new issue which you were working on #14 
image

**Comment by Saumya40-codes** [comment]
> imo this is part of the new issue which you were working on

We can go on with this, but make sure to make changes of different issue not on the same branch :)

**Comment by KushalXCoder** [comment]
While committing the last time, I, by mistake, did git add . instead of a specific file, and hence, two issues got pushed. But I will take care of it the next time.
But thanks for accepting the PR :)

**Pull request merged**